File: src/tool/spring-boot-generator/output/test/db/curveInfo.js
let curveInfo = {
  name: "curveInfo",
  description: "曲线信息",
  columns: {
    curveId: {
      name: "curveId",
      description: "曲线编号",
      isPk: true,
      autoIncrement: true,
      nullable: false,
      type: {
        name: "int",
        length: null
      }
    },
    trackDivisionId: {
      name: "trackDivisionId",
      description: "宸ュ姟娈电紪鍙?",
      nullable: true,
      type: {
        name: "int",
        length: null
      }
    },
    lineId: {
      name: "lineId",
      description: "线路编号",
      nullable: true,
      type: {
        name: "int",
        length: null
      }
    },
    side: {
      name: "side",
      description: "线别",
      nullable: true,
      type: {
        name: "int",
        length: null
      }
    },
    direction: {
      name: "direction",
      description: "行别",
      nullable: true,
      type: {
        name: "int",
        length: null
      }
    },
    startMiles: {
      name: "startMiles",
      description: "起点里程",
      nullable: true,
      type: {
        name: "float"
      }
    },
    endMiles: {
      name: "endMiles",
      description: "缁堢偣閲岀▼",
      nullable: true,
      type: {
        name: "float",
        length: null
      }
    },
    curveDirection: {
      name: "curveDirection",
      description: "鏇茬嚎鏂瑰悜",
      nullable: true,
      type: {
        name: "int",
        length: null
      }
    },
    curveReadius: {
      name: "curveReadius",
      description: "鏇茬嚎鍗婂緞",
      nullable: true,
      type: {
        name: "float"
      }
    },
    steeringAngle: {
      name: "steeringAngle",
      description: "转向角",
      nullable: true,
      type: {
        name: "float"
      }
    },
    trackDisType: {
      name: "trackDisType",
      description: "轨距类型",
      nullable: true,
      type: {
        name: "int",
        length: null
      }
    },
    trackDisWide: {
      name: "trackDisWide",
      description: "轨距加宽",
      nullable: true,
      type: {
        name: "float"
      }
    },
    overHeight: {
      name: "overHeight",
      description: "超高",
      nullable: true,
      type: {
        name: "float"
      }
    },
    slopeRate: {
      name: "slopeRate",
      description: "顺坡率",
      nullable: true,
      type: {
        name: "float"
      }
    },
    startTangentLength: {
      name: "startTangentLength",
      description: "起点切线长",
      nullable: true,
      type: {
        name: "float"
      }
    },
    endTangentLength: {
      name: "endTangentLength",
      description: "终点切线长",
      nullable: true,
      type: {
        name: "float"
      }
    },
    startSlowLength: {
      name: "startSlowLength",
      description: "起缓和线长",
      nullable: true,
      type: {
        name: "float"
      }
    },
    endSlowLength: {
      name: "endSlowLength",
      description: "终缓和线长",
      nullable: true,
      type: {
        name: "float"
      }
    },
    curveLength: {
      name: "curveLength",
      description: "鏇茬嚎闀?",
      nullable: true,
      type: {
        name: "float"
      }
    },
    averageSpeed: {
      name: "averageSpeed",
      description: "平均速度",
      nullable: true,
      type: {
        name: "float"
      }
    },
    longChain: {
      name: "longChain",
      description: "长链标记",
      nullable: true,
      type: {
        name: "int",
        length: null
      }
    },
    remark: {
      name: "remark",
      description: "备注",
      nullable: true,
      type: {
        name: "varchar",
        length: null
      }
    },
    status: {
      name: "status",
      description: "状态",
      nullable: true,
      type: {
        name: "int",
        length: null
      }
    },
    isDelete: {
      name: "isDelete",
      description: "删除标记",
      nullable: false,
      type: {
        name: "int",
        length: null
      }
    },
    createTime: {
      name: "createTime",
      description: "创建时间",
      nullable: true,
      type: {
        name: "datetime"
      }
    },
    updateTime: {
      name: "updateTime",
      description: "更新时间",
      nullable: true,
      type: {
        name: "datetime"
      }
    },
    operator: {
      name: "operator",
      description: "操作人",
      nullable: true,
      type: {
        name: "int",
        length: null
      }
    }
  }
};

let columnsArray = [
  "curveId",
  "trackDivisionId",
  "lineId",
  "side",
  "direction",
  "startMiles",
  "endMiles",
  "curveDirection",
  "curveReadius",
  "steeringAngle",
  "trackDisType",
  "trackDisWide",
  "overHeight",
  "slopeRate",
  "startTangentLength",
  "endTangentLength",
  "startSlowLength",
  "endSlowLength",
  "curveLength",
  "averageSpeed",
  "longChain",
  "remark",
  "status",
  "isDelete",
  "createTime",
  "updateTime",
  "operator"
];

function create() {
  return {
  curveId: 14,
  trackDivisionId: 232,
  lineId: 190,
  side: 108,
  direction: 965,
  startMiles: 63,
  endMiles: 560,
  curveDirection: 83,
  curveReadius: 212,
  steeringAngle: 435,
  trackDisType: 250,
  trackDisWide: 778,
  overHeight: 441,
  slopeRate: 24,
  startTangentLength: 206,
  endTangentLength: 27,
  startSlowLength: 974,
  endSlowLength: 925,
  curveLength: 363,
  averageSpeed: 541,
  longChain: 929,
  remark: "备注282",
  status: 473,
  isDelete: 577,
  createTime: "2020-1-23 1:3:35",
  updateTime: "2020-1-23 1:3:35",
  operator: 223
};
}

curveInfo.columnsArray=columnsArray;
curveInfo.create=create;
module.exports=curveInfo;

File: src/tool/spring-boot-generator/output/railway/railway/src/main/java/com/railway/pojo/entity/CurveInfo.java
package com.railway.pojo.entity;

import io.swagger.annotations.ApiModel;
import io.swagger.annotations.ApiModelProperty;
import lombok.Getter;
import lombok.Setter;

import java.util.Date;

@ApiModel("曲线信息")
@Getter
@Setter
public class CurveInfo  {

    @ApiModelProperty("平均速度")
    private Float averageSpeed;

    @ApiModelProperty("创建时间")
    private Date createTime;

    @ApiModelProperty("鏇茬嚎鏂瑰悜")
    private Integer curveDirection;

    @ApiModelProperty("曲线编号")
    private Integer curveId;

    @ApiModelProperty("鏇茬嚎闀?")
    private Float curveLength;

    @ApiModelProperty("鏇茬嚎鍗婂緞")
    private Float curveReadius;

    @ApiModelProperty("行别")
    private Integer direction;

    @ApiModelProperty("缁堢偣閲岀▼")
    private Float endMiles;

    @ApiModelProperty("终缓和线长")
    private Float endSlowLength;

    @ApiModelProperty("终点切线长")
    private Float endTangentLength;

    @ApiModelProperty("删除标记")
    private Integer isDelete;

    @ApiModelProperty("线路编号")
    private Integer lineId;

    @ApiModelProperty("长链标记")
    private Integer longChain;

    @ApiModelProperty("操作人")
    private Integer operator;

    @ApiModelProperty("超高")
    private Float overHeight;

    @ApiModelProperty("备注")
    private String remark;

    @ApiModelProperty("线别")
    private Integer side;

    @ApiModelProperty("顺坡率")
    private Float slopeRate;

    @ApiModelProperty("起点里程")
    private Float startMiles;

    @ApiModelProperty("起缓和线长")
    private Float startSlowLength;

    @ApiModelProperty("起点切线长")
    private Float startTangentLength;

    @ApiModelProperty("状态")
    private Integer status;

    @ApiModelProperty("转向角")
    private Float steeringAngle;

    @ApiModelProperty("轨距类型")
    private Integer trackDisType;

    @ApiModelProperty("轨距加宽")
    private Float trackDisWide;

    @ApiModelProperty("宸ュ姟娈电紪鍙?")
    private Integer trackDivisionId;

    @ApiModelProperty("更新时间")
    private Date updateTime;

}
